spec/system/consumer/caching/shops_caching_spec.rb

```diff
@@ -14,9 +14,12 @@
   }
 
   describe "caching enterprises AMS data" do
-    xit "caches data for all enterprises, with the provided options" do
-      pending "#11010"
+    it "caches data for all enterprises, with the provided options" do
+      # Trigger lengthy tasks like JS compilation before testing caching:
+      visit shops_path
```

Comment:
Just wondering what this first visit is for?

Reply:
Oh, I was hoping that the comment would explain that. During the first visit the JS assets are compiled. So when we get to the second visit, it doesn't take ages to render the page and we can assert on the cache key before it expires.

Reply:
Ohhh, sorry yes that's clearer now. I misunderstood and thought the `Rails.cache.clear` was to clear the JS compilation to make the second call intentionally slower. But I guess that cache clear is relating to the caching test.